ruby/openproject.nix

```diff
@@ -120,6 +151,8 @@ in
       hash = npmDepsHash;
     };
 
+    makeCacheWritable = true;
```

Comment:
I got an npm error without this

Reply:
Yeah, unfortunately this sort of thing is quite common with npm builds.